src/components/OutlinePopup.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ComponentProps, ReactNode } from 'react';
+import styled, { css } from 'styled-components';
+import { Popup } from '@taskany/bricks';
+import { radiusM, gapS, gray5 } from '@taskany/colors';
+
+const arrowDirectionStyles = {
```

Comment:
I'm not sure we need this component. Popup has prop to show and hide arrow. But here in Dropdown arrow must be visible.